src/ztex/ztex.c

```diff
@@ -335,6 +315,7 @@ struct ztex_device *ztex_find_by_sn(struct ztex_dev_list *dev_list, char *sn)
 	}
 	return NULL;
 }
+*/
```

Comment:
Let's use `#if 0` ... `#endif` rather than "comments" for excluding pieces of code.

Reply:
OK, I see a rationale of ```#if 0 ... #endif``` - the construct can be nested.